workflow-report/README.md

```diff
@@ -0,0 +1,44 @@
+# Github Action: report a failed workflow to VK Team chat
+This action sends the following information to VK Team chat:
+* __Job__: name of the failed job with a link to this job
+* __Commit__: hash of the commit started the job with a link to this commit
+* __Branch__: name of the branch
```

Comment:
name of the branch with the link to this branch

Reply:
Done